app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
     <string name="app_name">VenueNext SDK Demo</string>
-    <string name="app_version">0.8.4</string>
+    <string name="app_version">0.9.0</string>
```

Comment:
(Aside) We should consider exposing our SDK version so integrators can maybe display it somewhere in their apps 👀

Reply:
I like that.

Reply:
Maybe let’s ask product about doing something like this? Even if at the very least we can get them to add it in a settings page or something, that’d be cool.